Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEvents.java

```diff
@@ -1389,6 +1389,11 @@ public void onFade(BlockFadeEvent event) {
             case BUBBLE_CORAL_FAN:
             case FIRE_CORAL_FAN:
             case HORN_CORAL_FAN:
+            case BRAIN_CORAL_WALL_FAN:
+            case BUBBLE_CORAL_WALL_FAN:
+            case FIRE_CORAL_WALL_FAN:
+            case HORN_CORAL_WALL_FAN:
+            case TUBE_CORAL_WALL_FAN:
```

Comment:
If we're worried about entity types causing issues, might this also be something we're worried about?

Reply:
What do you mean? This is not related to 1.16.

Reply:
New ones shouldn't cause issues. The issue is when removing enums, as older server versions won't get the same level of protection

Reply:
That makes sense

Reply:
These were added to Minecraft awhile ago so it shouldn't cause any compatibility problems.